apps/system/lockscreen/js/lockscreen_state_manager.js

```diff
@@ -542,10 +564,13 @@
           this.onUnlock();
         }
         break;
+      case 'lockscreen-appopened':
       case 'lockscreen-request-lock':
+        this.resumeEvents(['holdcamera']);
```

Comment:
Since in the scenario we need to suspend multiple events, we can call it in a loop, and push into the inner array of suspending events one by one.